client/src/components/Page/CreatedRunPage.js

```diff
@@ -8,6 +8,11 @@ import { useParams } from "react-router-dom";
 const CreatedRunPageContainer = styled.div`
   width: 100%;
   height: 100%;
+  margin-top: 30%;
+  margin-bottom: 15%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
```

Comment:
Looks like this container is basically the same as the ChooseYourRunPageContainer – you could just use this and update with the additional styling :)

Reply:
Good Idea. Thanks